light_node/src/configuration.rs

```diff
@@ -622,6 +624,13 @@ pub fn tezos_app() -> App<'static, 'static> {
             .value_name("NUM")
             .help("Number of threads spawned by a tokio thread pool. If value is zero, then number of threads equal to CPU cores is spawned.")
             .validator(parse_validator_fn!(usize, "Value must be a valid number")))
+        .arg(Arg::with_name("maindb-backend")
+            .long("maindb-backend")
+            .takes_value(true)
+            .value_name("STRING")
+            .possible_values(&TezedgeDatabaseBackendConfiguration::possible_values())
+            .default_value("sled")
```

Comment:
@mambisi 
whichone is default sled or rocksdb,
here is sled and below is rocksdb